org.adempiere.base/src/org/compiere/model/GridTable.java

```diff
@@ -2425,9 +2425,22 @@ else if (value instanceof Timestamp) {
 					multiRowWHERE.append (tableName).append(".").append(columnName)
 						.append ("=").append (DB.TO_STRING(value.toString()));
 			}
+			if (columnName.equals(PO.getUUIDColumnName(tableName)))

```

Comment:
so the design here is the table with UUID key column wouldn't mark that UUID column as IsKey=Y ?

Reply:
Right - IsKey would be reserved just for ID keys - I changed the DisplayLogic of that field to be shown just when the reference=ID (and when is already set for some exception cases that are already present in core)

Reply:
ic, this part doesn't looks right, it shouldn't happens for every table with _ID and _UU column.